### PR TITLE
Correct directory for versions.yml file

### PR DESCRIPTION
**Pull request by danhermann**
Resolves https://github.com/logstash-plugins/logstash-input-java_input_example/issues/6

### CODE REVIEW

File: rubyUtils.gradle [hunk 1]
```diff
@@ -183,7 +183,7 @@ task downloadJRuby(type: Download) {
     description "Download JRuby artifact from this specific URL: ${jRubyURL}"
     src jRubyURL
     onlyIfNewer true
-    inputs.file("${projectDir}/versions.yml")
+    inputs.file("${versionsPath}/versions.yml")
```
Comment (jsvd) [inline rubyUtils.gradle]: versionsPath already contains the yaml file, so we can just do:

```suggestion
    inputs.file(versionsPath)
```

Comment (danhermann) [reply rubyUtils.gradle]: Thanks, @jsvd and @andsel. You're right about the path and I've updated the PR accordingly.

**Comment by danhermann** [comment]
Jenkins, test this please

**Comment by danhermann** [comment]
Unrelated test failure

**Comment by jsvd** [comment]
Many thanks for this @danhermann <3

**Pull request closed without merging**